contracts/WildcardsQV.sol

```diff
@@ -41,83 +61,113 @@ contract WildcardsQV is Initializable {
     ////////////////////////////////////
     //////// SETUP CONTRACT////////////
     //// NOTE: Upgradable at the moment
-    function initialize(uint256 _votingInterval) public initializer {
+    function initialize(uint256 _votingInterval, ILoyaltyToken _addressOfLoyalyTokenContract, IWildCardToken _addressOfWildCardTokenContract, IWildCardSteward _addressOfWildCardStewardContract, address _addressOfDragonCard) public initializer {
         admin = msg.sender;
         votingInterval = _votingInterval;
 
         proposalDeadline = now.add(_votingInterval);
+
+        loyaltyToken = _addressOfLoyalyTokenContract;
+        wildCardToken = _addressOfWildCardTokenContract;
+        wildCardSteward = _addressOfWildCardStewardContract;
+        addressOfDragonCard = _addressOfDragonCard;
     }
 
     ///////////////////////////////////
     /////// Config functions //////////
     ///////////////////////////////////
     function changeVotingInterval(uint256 newInterval) external onlyAdmin {
         votingInterval = newInterval;
+        emit LogNewVotingInterval(newInterval);
+    }
+
+    function changeProposalState(uint256 _proposalId, bool newState) external onlyAdmin {
```

Comment:
Great! We should think more about when we want to put a proposal into the withdrawn state. Maybe at some point some of this logic gets automated in the increaseIteration function (later down the line I guess)